api-reference/50 Common/Object Structures/dxSchedulerTimeZone/title.md

```diff
@@ -4,7 +4,7 @@ type: String
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A formatted time zone representation. Includes the **id** and **offset** values.
```

Comment:
Why 'representation'?

Reply:
It's not a time zone, but a formatted string that contains a time zone id and its offset. E.g.: title: "(GMT +02:00) Europe/Berlin"

Reply:
"Time zones are often represented by alphabetic abbreviations such as "EST", "WST", and "CST"

"Offsets from UTC/GMT are written in the format ±[hh]:[mm], ±[hh] [mm], or ±[hh] (either hours ahead or behind UTC/GMT). For example, if the time being described is one hour ahead of UTC (such as the time in Berlin during the winter), the zone designator would be "+01:00", "+0100", or simply "+01"."

It does not seem correct to call it "A formatted time zone representation" because 'a formatted representation' can mean different things.